### DISCUSSION TITLE
i don't like liquid oxygenite

### DISCUSSION THREAD

**Discussion by LePiromano** [FakeFishGames/Barotrauma#13558] [Ideas and suggestions]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

liquid oxygenite is a drug that has been in the game for a long-ass time as an alternative to CPR, it was supposed to be the safer and quicker alternative but with the 50 skill requirement and the massive amounts of organ damage it did back then, little to no medics used it at all, now CPR was nerfed a bit and just when you think liquid oxygenite might shine again adrenaline gets its new use, and now liquid oxygenite is not more than a crafting recipe item (which is by the way not communicated at all in the description and it uses the medical syringe sprite instead of the chemical bottle sprite from opium/adrenaline).
i've been told it's a good explosive weapon too but with the damage it does you might as well be throwing flash powder at their feet
i don't like it because, for an item that comes from a late-game alien material, is severely underpowered unless you're trying to mass  produce something like deusizine

there's a million things you could do but if you want an idea, just add a new affliction that decreases the amount of oxygen you drain from suits/masks while you have it and give it to liquid oxygenite, i dunno get creative

### Reproduction steps

-use liquid oxygenite
-be dissappointed
-break down on the floor and cry

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.2.13.0 (unstable)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Yeah, I agree in the sense that it's not that useful as a drug, it mainly just serves as a material for crafting other items. We might look into giving it some more use in the future, but I'll close the ticket regardless, as this isn't really a bug report.

**Reply by VaelophisNyx** [reply]
a recommendation would be to make it a temporary oxygen restore that can exceed normal maximum O2 in the body (you can go longer without an air tank, roughly is the idea)
this would give it utility on long dives and flooded sub conditions
